src/main/java/ru/hogwarts/school/controller/AvatarController.java

```diff
@@ -0,0 +1,27 @@
+package ru.hogwarts.school.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import ru.hogwarts.school.model.Avatar;
+import ru.hogwarts.school.service.StudentService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("avatars")
```

Comment:
```suggestion
@RequestMapping("/avatars")
```
слэш в начале лучше писать